src/zzz_od/application/matrix_action/matrix_action_app.py

```
@@ -0,0 +1,612 @@
+    def _round_success_with_click_reset(self, status: Optional[str] = None) -> OperationRoundResult:
+        self._reset_click_cooldown()
+        return self.round_success(status)
+
+    def _reset_click_cooldown(self) -> None:
+        self._last_click_at = 0.0
+
+    def _is_click_on_cooldown(self) -> bool:
+        return time.monotonic() - self._last_click_at < self._click_cooldown_sec
+
+    def _click_with_cooldown(self, pos) -> bool:
+        # 先记录点击尝试时间，避免底层点击返回失败时短时间内重复点击同一目标
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)